tests/test_data/packages/fake_with_deps/setup.py

```diff
@@ -18,5 +18,6 @@
         "SQLAlchemy!=0.9.5,<2.0.0,>=0.7.8,>=1.0.0",
         "python-memcached>=1.57,<2.0",
         "xmltodict<=0.11,>=0.4.6",
+        "requests @ git+git://github.com/psf/requests@v2.25.1",
```

Comment:
This new dependency does not bring anything more by itself, so I could either:

1. remove it
2. assert inside `test_realistic_complex_sub_dependencies` that `out.stderr` contains a direct reference